Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml

```diff
@@ -1467,7 +1467,13 @@ script:
 
             var fields = verifyOrMakeArray(record.Field);
             fields.forEach(function (field) {
-                toContext.Fields[GLOBAL_MAPPING[field['-id']].Name] = field['#text'];
+                if (field['#text'] != null) {
+                    toContext.Fields[GLOBAL_MAPPING[field['-id']].Name] = field['#text'];
+                } else if (field.ListValues != null && field.ListValues.ListValue['#text'] !== null) {
+                    toContext.Fields[GLOBAL_MAPPING[field['-id']].Name] = field.ListValues.ListValue['#text'];
+                } else {
+                    toContext.Fields[GLOBAL_MAPPING[field['-id']].Name] = field['#text'];
```

Comment:
replacee to null please

Reply:
I rather not, could be undefined. Critical fin your opinion?